### PR TITLE
API discrepancy + pipeline + doc fixes - Closes #424

### LINKED ISSUE #424: Minor API updates
### Description

Consider the following API updates:

- Rename `/api/v2/transactions?block=` param to  `/api/v2/transactions?blockId=`
- Document all available params of `/api/v2/accounts`

Moved: 

- Review transaction statistics - they are not being updated on CD - moved to #433

### Motivation

- For API consistency it's better to keep the same naming convention in all endpoints

### Acceptance Criteria

- API definitions are updated
- Swagger specs are up-to-date
- Docs `docs/api/version2.md` is up-to-date
- Tests are up-to-date and passing

### PR DESCRIPTION
**Pull request by MichalTuleja**
### What was the problem?
This PR resolves #424

### How was it solved?

- [x] Updated block parameter (Merged in #456)
- [x] Update response schema (Merged in #456)
- [x] Update tests (Merged in #456)
- [x] Update documentation
- [x] Minor refactor (Merged in #456)

### How was it tested?

Jenkins

### CODE REVIEW

**Review by nagdahimanshu — APPROVED** [review]
LGTM!

File: docs/api/version2.md [hunk 1]
```diff
@@ -530,9 +538,13 @@ _Supports pagination._
 | recipientAddress   | String  | `/^lsk[a-hjkm-z2-9]{38}$//^[1-9]\d{0,19}[L\|l]$/`                  | *(empty)*      |
 | recipientPublicKey | String  | `/^([A-Fa-f0-9]{2}){32}$/`                                         | *(empty)*      |
 | recipientUsername  | String  | `/^[a-z0-9!@$&_.]{1,20}$/`                                         | *(empty)*      |
-| data               | String  |                                                                    | *(empty)*      | Wildcard search ie. "gene*" -> "genesis"                       |
+search ie. "gene*" -> "genesis"                       |
```
Comment (sameersubudhi) [inline docs/api/version2.md:541]: Seems inconsistent.

**Review by sameersubudhi — CHANGES_REQUESTED** [review]

File: docs/config_options.md [hunk 2]
```diff
@@ -4,6 +4,10 @@
 
 These options are available for all micro-services provided by Lisk Service.
 
+Note that most of them are set properly in the Docker environment.
```
Comment (chris529) [inline docs/config_options.md:7]:
```suggestion
Note that most of them are set correctly in the Docker environment.
```

File: docs/config_options.md [hunk 3]
```diff
@@ -4,6 +4,10 @@
...
+
+If the Docker environment is being used, the only required variable `LISK_CORE_WS` is needed.
```
Comment (chris529) [inline docs/config_options.md:9]:
```suggestion
If the Docker environment is being used, only the required variable `LISK_CORE_WS` is needed.
```

**Review by chris529 — COMMENTED** [review]
Hi Michal, Looks very good, just a couple of very small points. 👍

**Review by sameersubudhi — APPROVED** [review]

**Pull request merged**